### ISSUE TITLE
Add collaborators as "Read" to this repo

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/admin#517]
@nodejs/collaborators or @nodejs/members should be added so they can participate in locked conversations.

**Comment by tniessen** [comment]
+1 to @nodejs/members.

**Comment by codeekage** [comment]
+1

**Comment by tniessen** [comment]
Adding @nodejs/members with "Read" access might not help:

Citing @bnb from https://github.com/nodejs/admin/issues/516#issuecomment-650473171:

> only people with write access to this repository can now comment, which excludes much of the project's collaborators.

[GitHub seems to confirm this.](https://help.github.com/en/github/building-a-strong-community/locking-conversations)

**Comment by mmarchini** [comment]
Any concerns on adding @nodejs/members with write permission? If we want to limit commits here, we could set branch restrictions.

**Comment by gireeshpunathil** [comment]
I would rather move the relevant issue ( #516) to a less restricted repo such as core. Adding write permissions to the org members does not seem to align well with the stated purpose of this repo? (`Administration for TSC + Community Committee collaborative efforts`)

**Comment by bnb** [comment]
>I would rather move the relevant issue ( #516) to a less restricted repo such as core.

This would exclude CommComm from the discussion.

**Comment by tniessen** [comment]
We should probably just unlock #516 then, and delete undesired comments quickly.

**Comment by gireeshpunathil** [comment]
> This would exclude CommComm from the discussion.

apologies, did not know that, and definitely did not mean it!

**Comment by bnb** [comment]
>apologies, did not know that, and definitely did not mean it!

no offense taken! Just wanted to make sure that it was shared <3

**Comment by mmarchini** [comment]
I agree with @tniessen, and if the volume of undesired comments grows quickly, we can lock again and figure out another way to handle this.
